CHANGELOG.md

```diff
@@ -9,6 +9,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
+## [0.11.1] - 2022-03-18
+
+## Highlights
+
+Major changes include:
+* A number of changes to `fix` code to make these more robust
+* Improvements to templating blocks
+* `generate_parse_fixture_yml` options to allow quicker, partial regeneration of YML files
+* Numerous rule fixes including adding auto fix to L042
+* Numerous grammar changes
+
+## What's Changed
+
+* Spark3: Support for `SHOW` statements [#2864](https://github.com/sqlfluff/sqlfluff/pull/2864) [@R7L208](https://github.com/R7L208)
+* Add Markerr to list of organizations using SQLFluff in the wild  [#2874](https://github.com/sqlfluff/sqlfluff/pull/2874) [@kdw2126](https://github.com/kdw2126)
```

Comment:
```suggestion
* Add Marker to list of organizations using SQLFluff in the wild  [#2874](https://github.com/sqlfluff/sqlfluff/pull/2874) [@kdw2126](https://github.com/kdw2126)
```

Reply:
Nope, apparently it's Markerr: https://www.markerr.com

Reply:
Oh wow! It _looked_ like a typo, heh.